src/index.html

```diff
@@ -1,12 +1,401 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a class="logo-link" href="index.html">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav-menu">
```

Comment:
```suggestion
      <nav class="nav">
```

Reply:
try to simplify class names, removing '-menu' will not change something so remove it